### ISSUE TITLE
[capture-col] more intuitive error when no groups are captured

### ISSUE THREAD

**Issue by geekscrapy** [saulpw/visidata#1032]
Currently if you forget to add a capture group to the regex an error is shown in the status bar `IndexError: tuple index out of range` which isn't overly intuitive. Maybe a couple of tests could be added to determine if there were any matches.

I'm unsure how this relates to the row sampling though...

Current error when trying to use a regex with no capture group.
```
Traceback (most recent call last):
  File "/Users/geekscrapy/Documents/pyv/py3.7/lib/python3.7/site-packages/visidata/threads.py", line 219, in _toplevelTryFunc
    t.status = func(*args, **kwargs)
  File "/Users/geekscrapy/Documents/pyv/py3.7/lib/python3.7/site-packages/visidata/regex.py", line 66, in addRegexColumns
    vs.addColumnAtCursor(*cols.values())
  File "/Users/geekscrapy/Documents/pyv/py3.7/lib/python3.7/site-packages/visidata/sheets.py", line 506, in addColumnAtCursor
    return self.addColumn(*cols, index=index)
  File "/Users/geekscrapy/Documents/pyv/py3.7/lib/python3.7/site-packages/visidata/sheets.py", line 498, in addColumn
    return cols[0]
IndexError: tuple index out of range
```

TLDR: Maybe a small change of the `capture-col` prompt from `match regex:` to something like `capture regex:` would also assist? I appreciate that this ties back to the regex capture methods, but maybe more intuitive? Also the same with `select-col-regex` prompt, `select regex:` to `match regex:`. I'm in two minds if this would be confusing or not to experienced users. On one hand they tie into the regex methods, however the long commands dont reflect the regex methods anyways, on the other hand, maybe the prompt should just be the long command string? 🤷‍♂️  #thoughts :)

**Comment by saulpw** [comment]
Thanks for the ideas, @geekscrapy.  I agree with changing `match` to `capture`, but I think `select` works better than `match` because it actually selects rows on the sheet.  We want to go through and audit all the input() messages sometime soon, and that would a good time to revisit some of the more ambitious ideas (like having all prompts just be the long command-string, like emacs effectively does).
